test/e2e/build_test.go

```diff
@@ -105,4 +107,39 @@ var _ = Describe("Podman build", func() {
 		session.WaitWithDefaultTimeout()
 		Expect(session.ExitCode()).To(Equal(0))
 	})
+
+	It("podman build Containerfile locations", func() {
+		// Given
+		// Switch to temp dir and restore it afterwards
+		cwd, err := os.Getwd()
+		Expect(err).To(BeNil())
+		Expect(os.Chdir(os.TempDir())).To(BeNil())
```

Comment:
@baude You're more familiar with the tests than I - would we expect this to cause any issues with parallel testing? I don't think so...